### PR TITLE
modifying test to make it generic to support all apps and volumes

### PR DESCRIPTION
**Pull request by vprabhakar-px**
fixing test script to handle all apps with storage provisioners

### CODE REVIEW

File: tests/basic/pure_test.go [hunk 1]
```diff
@@ -202,15 +204,44 @@ var _ = Describe("{BringUpLargePodsVerifyNoPanic}", func() {
 			}
 		}()
 
-		Inst().AppList = []string{"nginx-fa-davol"}
+		// Inst().AppList = []string{"nginx-fa-davol"}
```
Comment (lsrinivas-pure) [inline tests/basic/pure_test.go:207]: delete if not required

**Review by lsrinivas-pure — APPROVED** [review]
minor comment

File: tests/basic/pure_test.go (hunk 1 above)
Comment (vprabhakar-px) [reply tests/basic/pure_test.go]: done , deleted

File: tests/basic/pure_test.go [hunk 2]
```diff
@@ -139,39 +139,41 @@ var _ = Describe("{PureFACDTopologyValidateDriveLocations}", func() {
...
+				1. Deploying nginx pods using two FADA volumes in 125 name-space simultaneously
+				2. After that verify if any panic in the logs due to nil pointer deference.
 	*/
 	var testrailID = 0
 	var runID int
 	JustBeforeEach(func() {
 		StartTorpedoTest("BringUpLargePodsVerifyNoPanic",
-			"Px should not panic when large number of pools are created", nil, testrailID)
+			"Validate no panics when creating more number of pods on "+
+				"FADA/Generic Volumes while kvdb failover in progress", nil, testrailID)
```
Comment (Santhoshairody) [inline tests/basic/pure_test.go:160]: Please test with FB volumes as well.

File: tests/basic/pure_test.go [hunk 3]
```diff
@@ -261,6 +292,7 @@ var _ = Describe("{BringUpLargePodsVerifyNoPanic}", func() {
 		// Waiting for all pods to become ready and in running state
 		waitForPodsRunning := func() (interface{}, bool, error) {
 			for _, eachContext := range contexts {
+				log.Infof("Verifying Context [%v]", eachContext.App.Key)
 				err := Inst().S.WaitForRunning(eachContext, 5*time.Minute, 2*time.Second)
```
Comment (Santhoshairody) [inline tests/basic/pure_test.go:296]: DOn't we need to check all volume state is up?

**Pull request merged**